### DISCUSSION TITLE
STOP QR CODE REGENERATION

### DISCUSSION THREAD

**Discussion by hamisirizwan** [WhiskeySockets/Baileys#400] [General Questions]
How can i enable maximum qr code tries?. coz it keeps generating qr code even for a connection that is nolonger needed. I want when it generates qrcode 3 times and its not scanned, it stops

**Comment by Riders004** [comment]
Can you explain more briefly

**Comment by hamisirizwan** [comment]
when the qr code shows and it is not scanned, it keeps regenartion to infinity, i want it to stop generating when it appears 3 times and not scanned @Riders004

**Reply by Riders004** [reply]
It's not generate infinity after some it's stopped

**Comment by hamisirizwan** [comment]
after how long and how can i limit that if you dont scan the qr code it stops generating?

**Reply by Riders004** [reply]
I don't but if don't wanna use qr method then use Pairing code method

**Comment by hamisirizwan** [comment]
how does the pairing code works?

**Reply by Riders004** [reply]
Just wrote your number after its given code out in your linked section
